src/core/cli/stop.go

```
@@ -480,276 +392,171 @@ func stopAllAgents(pm *PIDManager, timeout time.Duration, force, quiet bool) err
+					displayName, len(groups), formatGroups(groups))
 			}
+			return nil
 		}
-	}
 
-	// Stop registry last (sequential)
-	if registry != nil {
+		// Only stop the service if its PID file is present and the process is alive.
+		info, err := lifecycle.LookupService(service)
+		if err != nil || info == nil {
+			return nil
```

golangci-lint (2.11.4)

[error] error is not nil (line 480) but it returns nil

(nilerr)